src/ocean/text/convert/Formatter_test.d

```diff
@@ -559,3 +559,34 @@ unittest
     // Used to work only with "{:X}", however this limitation was lifted
     assert(format("{X}", 42) == "2A");
 }
+
+
+/*******************************************************************************
+
+    unit tests for `Range`
+
+*******************************************************************************/
+
+unittest
```

Comment:
Can you add an unittest with:
```
struct Key { int value; }
struct Value { string value; }
Key[Value] aa;
aa.byKeyValue;
```

And some data in it (4 or more elements) ?

Reply:
I just added it.